django_project/lesson/models/answer.py

```diff
@@ -56,5 +58,12 @@ class Meta:
         app_label = 'lesson'
         unique_together = ['answer_number', 'question']
```

Comment:
Do we need to do this? When first entering an answer initially we will either need to ensure that answers are given a unique sequence number or remove this constraint. I probably prefer to remove this constraint and implement sortable behaviour on the question list.

Reply:
Sorry, I fail to understand. Whatever if we do the sortable behavior or not, we can't have two answers having the same number and belonging to the same question.

>  I probably prefer to remove this constraint and implement sortable behaviour on the question list.

I'm missing something. We need to keep this uniqueness, especially if we do the sortable behavior, we won't have duplicates.